src/posts/owa-eu-dma-submission-apple-ipados.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'OWA submits response to EU DMA investigation into Apple iPadOS'
+date: '2023-11-21'
+tags: ['Policy', 'Updates', 'EU']
+author: "Frances Berriman"
+---
+
+As the EU further rolls out the Digital Marketing Act, they have designated a variety of companies and their platforms as “Gatekeepers”. You can see the [complete list of Gatekeepers](https://ec.europa.eu/commission/presscorner/detail/en/ip_23_4328) and some really useful explainers on what being a [Gatekeeper means on the EU’s very informative website](https://commission.europa.eu/strategy-and-policy/priorities-2019-2024/europe-fit-digital-age/digital-markets-act-ensuring-fair-and-open-digital-markets_en).
+
+Now that Gatekeepers have been designated, the companies that operate those services have had the opportunity to rebut their designations. 
+
+A particularly interesting response was from Apple, who have made the case that iPadOS is a separate operating system entirely from iOS, both of which are currently designated as platforms subject to the DMA, along with the Apple AppStore. Their arguments essentially boil down to usage statistics for iPadOS alone being below the required usage numbers to fall under the rules and should not be designated a Gatekeeper platform service.
```

Comment:
`"is a separate operating system entirely from iOS"`

Shouldn't the `entirely` go before `separate`:

`"is an entirely separate operating system from iOS"`?

(Not a native speaker, so plz ignore  if this is actually correct :))

Reply:
Both technically work, but your suggestion reads better, so I'm changing it to that.